### PR TITLE
WIP: Implementing Tests

### PR DESCRIPTION
**Pull request by SomeoneWeird**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,7 @@ node_modules
 
 lib
 >>>>>>> vaultex-private/release
```
Comment (joshgillies) [inline .gitignore]: remnant of a merge conflict?

Comment (SomeoneWeird) [inline .gitignore]: indeed, will fix

File: test/vaultex.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+describe('Vaultex', function () {
+  before(function (done) {
+    this.timeout(100000)
+    spawnDevVault.download(process.cwd(), function (err) {
+      assert.ifError(err)
+      done()
+    })
+  })
+  it('should export fn', function () {
```
Comment (joshgillies) [inline test/vaultex.js:18]: care to throw a new line between this case and the next few. :smile:

**Comment by joshgillies** [comment]
lgtm :+1:

**Pull request closed without merging**